~/fabric8-analytics-common/dashboard/repositories/fabric8-analytics-ingestion ~/fabric8-analytics-common/dashboard/repositories/fabric8-analytics-ingestion
created virtual environment CPython3.6.10.final.0-64 in 148ms
  creator CPython3Posix(dest=/home/msawood/fabric8-analytics-common/dashboard/repositories/fabric8-analytics-ingestion/venv, clear=False, global=False)
  seeder FromAppData(download=False, pip=latest, setuptools=latest, wheel=latest, via=copy, app_data_dir=/home/msawood/.local/share/virtualenv/seed-app-data/v1.0.1)
  activators PythonActivator,FishActivator,XonshActivator,CShellActivator,PowerShellActivator,BashActivator
Requirement already satisfied: pyflakes in ./venv/lib/python3.6/site-packages (2.2.0)
----------------------------------------------------
Checking source files for common errors in following
directories:
f8a_ingestion
tests
alembic
tools
----------------------------------------------------

f8a_ingestion/models.py
    Pass
f8a_ingestion/__init__.py
    Pass
f8a_ingestion/rest_api.py
    Pass
f8a_ingestion/errors.py
    Pass
f8a_ingestion/enums.py
    Pass
f8a_ingestion/utils.py
    Pass
f8a_ingestion/defaults.py
    Pass
tests/__init__.py
    Pass
tests/conftest.py
    Pass
tests/test_errors.py
    Pass
tests/test_rest_api.py
    Pass
tests/test_utils.py
    Pass
alembic/versions/0194c3a2332e_initial_migration.py
    Pass
alembic/env.py
    Pass
tools/check_python_version.py
    Pass
----------------------------------------------------
Checking following source files for common errors:
setup.py
----------------------------------------------------

setup.py
    Pass
~/fabric8-analytics-common/dashboard/repositories/fabric8-analytics-ingestion
All checks passed for 16 source files
